library/core/src/ops/range.rs

```diff
@@ -52,7 +52,7 @@ impl fmt::Debug for RangeFull {
 /// A (half-open) range bounded inclusively below and exclusively above
 /// (`start..end`).
 ///
-/// The `Range` `start..end` contains all values with `x >= start` and
+/// The range `start..end` contains all values with `x >= start` and
 /// `x < end`. It is empty unless `start < end`.
```

Comment:
```suggestion
/// `start..end` contains all values with `start <= x` and `x < end`.
/// It is empty if `start >= end`.
```
`The range` feels redundant, probably can just remove it.

Also, I change the `unless` to `if` as I feel like it is harder to comprehend.

Reply:
It's good but I think it maybe be better to just remove `The range`, it is already talking about range.